### PR TITLE
Ft azure configuration

### PR DESCRIPTION
**Pull request by nodeJayS**
Azure Blobs and PostgreSQL configuration

### CODE REVIEW

File: .kubernetes/kubernetes_deployments.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
         ports:
         - containerPort: 80
         env:
-        - name: ConnectionStrings__default
+        - name: ConnectionString__default
```
Comment (escalonn) [inline .kubernetes/kubernetes_deployments.yaml:32]: this is incorrect. `GetConnectionString` method looks in the `ConnectionStrings` section.

Comment (escalonn) [reply .kubernetes/kubernetes_deployments.yaml:32]: ah, i see, you changed that. ok, as long as it's the same place the code looks

File: Fakebook.Posts/Fakebook.Posts.RestApi/Startup.cs [hunk 2]
```diff
@@ -27,21 +29,20 @@ public class Startup {
...
-            if (connectionString is null) {
-                throw new InvalidOperationException("No connection string 'defaualt' found.");
-            }
-
-            services.AddDbContext<FakebookPostsContext>(options =>
-            options.UseNpgsql(connectionString));
+            // setup Postgres database
+            if (Configuration["ConnectionString:default"] is string connectionString) 
+                services.AddDbContext<FakebookPostsContext>(options => options.UseNpgsql(connectionString));
+            else throw new NullReferenceException("No connection string 'default' found.");
```
Comment (escalonn) [inline Fakebook.Posts/Fakebook.Posts.RestApi/Startup.cs:35]: throwing nullreferenceexception is bad practice just fyi

Comment (harelfishbein) [reply Fakebook.Posts/Fakebook.Posts.RestApi/Startup.cs:35]: thanks, just read [Using Standard Exception Types](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/using-standard-exception-types) I'll change it to `ArgumentNullException`

**Pull request closed without merging**